services/core-web/src/components/mine/Permit/PermitConditionsSelectDocumentModal.tsx

```diff
@@ -0,0 +1,50 @@
+import { IPermitAmendmentDocument } from "@mds/common/interfaces";
+import React, { FC } from "react";
+import CoreTable from "@mds/common/components/common/CoreTable";
+import {
+  renderDateColumn,
+  renderTextColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import { Button, Typography } from "antd";
+import { documentNameColumn } from "@mds/common/components/documents/DocumentColumns";
+
+const { Title, Paragraph } = Typography;
+
+interface PermitConditionsSelectDocumentModalProps {
+  documents: IPermitAmendmentDocument[];
+  onSubmit: (document: IPermitAmendmentDocument) => void;
+}
+
+const PermitConditionsSelectDocumentModal: FC<PermitConditionsSelectDocumentModalProps> = ({
+  documents,
+  onSubmit,
+}) => {
+  return (
+    <div>
+      <Title>Extract Permit Conditions</Title>
+      <Paragraph>Select the permit for permit condition extraction.</Paragraph>
+      <CoreTable
+        rowKey="document_manager_guid"
+        dataSource={documents}
+        columns={[
+          documentNameColumn("document_name", "File Name"),
+          renderTextColumn("create_user", "Created By"),
+          renderDateColumn("create_timestamp", "Uploaded"),
+          {
+            key: "action",
+            render: (text, record: IPermitAmendmentDocument) => {
+              console.log(record);
```

Comment:
psst. Stray console log.
Also, fun fact: when there's no dataIndex on a column, the first argument of the render function is also the record. You should be able to take out "text" without anything changing